### PR TITLE
renamed namespace com.logstash to org.logstash

### PR DESCRIPTION
**Pull request by colinsurprenant**
all core tests are passing.

### CODE REVIEW

File: logstash-core-event-java/spec/event_spec.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 require "logstash/event"
 require "json"
 require "java"
-# java_import 'com.logstash.Util'
+# java_import 'Util'
```
Comment (talevy) [inline logstash-core-event-java/spec/event_spec.rb]: should we just remove this while we're at it?

File: logstash-core-event-java/src/main/java/org/logstash/bivalues/BiValues.java [hunk 2]
```diff
@@ -44,8 +44,8 @@
 
     private static HashMap<String, String> initCache() {
         HashMap<String, String> hm = new HashMap<>();
-        hm.put("com.logstash.Timestamp", "COM_LOGSTASH_TIMESTAMP");
-        hm.put("com.logstash.ext.JrubyTimestampExtLibrary$RubyTimestamp", "COM_LOGSTASH_EXT_JRUBYTIMESTAMPEXTLIBRARY$RUBYTIMESTAMP");
+        hm.put("Timestamp", "COM_LOGSTASH_TIMESTAMP");
```
Comment (talevy) [inline logstash-core-event-java/src/main/java/org/logstash/bivalues/BiValues.java]: did we want to remove the `xxx.logstash` prefixes here?

File: logstash-core-event-java/src/main/java/org/logstash/bivalues/BiValues.java [hunk 3]
```diff
@@ -12,14 +11,15 @@
 import org.jruby.ext.bigdecimal.RubyBigDecimal;
 import org.jruby.java.proxies.JavaProxy;
 import org.jruby.runtime.builtin.IRubyObject;
+import org.logstash.ext.JrubyTimestampExtLibrary;
```
Comment (talevy) [inline logstash-core-event-java/src/main/java/org/logstash/bivalues/BiValues.java]: why wasn't this changed to

```
import org.logstash.ext.JrubyTimestampExtLibrary.RubyTimestamp;
```

File: logstash-core-event-java/src/main/java/org/logstash/bivalues/TimestampBiValue.java [hunk 4]
```diff
@@ -1,14 +1,14 @@
-package com.logstash.bivalues;
+package org.logstash.bivalues;
 
-import com.logstash.Timestamp;
-import com.logstash.ext.JrubyTimestampExtLibrary.RubyTimestamp;
+import org.logstash.Timestamp;
 import org.jruby.Ruby;
+import org.logstash.ext.JrubyTimestampExtLibrary;
```
Comment (talevy) [inline logstash-core-event-java/src/main/java/org/logstash/bivalues/TimestampBiValue.java]: same as above, why no `.RubyTimestamp`?

File: logstash-core-event-java/src/test/java/org/logstash/AccessorsTest.java [hunk 5]
```diff
@@ -1,6 +1,8 @@
-package com.logstash;
+package org.logstash;
 
 import org.junit.Test;
+import org.logstash.Accessors;
```
Comment (talevy) [inline logstash-core-event-java/src/test/java/org/logstash/AccessorsTest.java]: why did this need to be added if it wasn't being used before?

File: logstash-core-event-java/src/test/java/org/logstash/EventTest.java [hunk 6]
```diff
@@ -1,6 +1,7 @@
-package com.logstash;
+package org.logstash;
 
 import org.junit.Test;
+import org.logstash.Event;
```
Comment (talevy) [inline logstash-core-event-java/src/test/java/org/logstash/EventTest.java]: why did this have to be added if it wasn't there before?

File: logstash-core-event-java/src/main/java/org/logstash/bivalues/BiValues.java (hunk 2 above)
Comment (colinsurprenant) [inline logstash-core-event-java/src/main/java/org/logstash/bivalues/BiValues.java]: good catch, probably not

**Comment by talevy** [comment]
LGTM

**Comment by guyboertje** [comment]
LGTM

**Comment by colinsurprenant** [comment]
merged in master, 5.0 and 5.x

**Pull request merged**